docs/website/content/v0.6/en/guides/upgrading/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Upgrading
+---
+
+In Kubernetes 1.19, the official `hyperkube` image was removed.
```

Comment:
should we add some note that only 0.5.1 -> 0.6 supported?

Reply:
That is more of a Talos upgrade thing. Kubernetes should run just fine no matter the upgrade order.

Reply:
Title of the page is `Upgrading` and applies only to `0.5.1` -> `0.6` for now, so I think it might be both Talos & Kubernetes, as Kubernetes is part of Talos in that regard

Reply:
In theory the upgrade of Talos and Kubernetes can independent to some a large degree. The ticket was only about adding a guide on how to upgrade Kubernetes, but I am happy to turn this into a guide for both. We do need it anyways. I will update the ticket.